src/AppInstallerRepositoryCore/PackageDependenciesValidation.cpp

```diff
@@ -82,23 +82,23 @@ namespace AppInstaller::Repository
         }
     
         void ThrowOnManifestValidationFailed(
-            std::vector<std::pair<DependentManifestInfo, Utility::Version>> failedManifests, std::string error)
+            std::vector<std::pair<DependentManifestInfo, Utility::Version>> failedManifests, AppInstaller::StringResource::StringId error)
         {
             auto itrStart = failedManifests.begin();
-            std::string dependentPackages{ itrStart->first.Id + "." + itrStart->first.Version };
+            std::vector<Manifest::ValidationError> validationErrors;
+            validationErrors.emplace_back(Manifest::ValidationError{error, "PackageIdentifier.PackageVersion", itrStart->first.Id + "." + itrStart->first.Version });
 
             std::for_each(
                 itrStart + 1,
                 failedManifests.end(),
                 [&](std::pair<DependentManifestInfo, Utility::Version> current)
                 {
-                    dependentPackages.append(", " + current.first.Id + "." + current.first.Version);
+                    validationErrors.emplace_back(Manifest::ValidationError{ error, "PackageIdentifier.PackageVersion", current.first.Id + "." + current.first.Version });
```

Comment:
I think if you are calling `emplace_back` you can omit the constructor and just pass the params. Like
`validationErrors.emplace_back(error, "PackageIdentifier.PackageVersion", current.first.Id + "." + current.first.Version);`